configs/next/packages/mpfr/sources

```diff
@@ -20,8 +20,8 @@
 #
 
 ATSRC_PACKAGE_NAME="GNU MPFR Library"
-ATSRC_PACKAGE_VER=4.0.1
-ATSRC_PACKAGE_REV=13327
+ATSRC_PACKAGE_VER=4.0.2
+ATSRC_PACKAGE_REV=13446
```

Comment:
OK.  The tag was created in revision 13435 which is contained in revision 13446.

```
$ svn log --stop-on-copy https://scm.gforge.inria.fr/anonscm/svn/mpfr/tags/4.0.2
------------------------------------------------------------------------
r13435 | vlefevre | 2019-01-31 20:40:43 -0200 (Thu, 31 Jan 2019) | 1 line

GNU MPFR 4.0.2 tag.
------------------------------------------------------------------------
```